src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4475,143 +4475,143 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
                               Visual C# Compiler Options
 
                         - OUTPUT FILES -
- /out:&lt;file&gt;                   Specify output file name (default: base name of
+ -out:&lt;file&gt;                   Specify output file name (default: base name of
```

Comment:
>- [](start = 1, length = 1)

It looks like now we can shift everything on space to the left. I Left that space there because ```/``` was interfering with doc comments in CSharpResources.Designer.cs

Reply:
After this PR the right column is still correctly aligned, and it is only separated from the left column by a single space in some lines. So I don't think we can shift the right column leftwards.
The change of `/` to `-` did not change alignment.

![image](https://user-images.githubusercontent.com/12466233/38443047-2f7e1ca0-399f-11e8-854a-f461c4d6f525.png)